website/cue/reference/components/sinks/elasticsearch.cue

```diff
@@ -226,11 +254,20 @@ components: sinks: elasticsearch: {
 		}
 		endpoint: {
 			description: "The Elasticsearch endpoint to send logs to. This should be the full URL as shown in the example."
-			required:    true
+			warnings: ["This option has been deprecated, the `endpoints` option should be used instead."]
+			required: true
```

Comment:
💭 should this still be required? I guess theoretically they're both "optional" from cue's standpoint, it's awkward.

In terms of site rendering I'd expect the better UX would be to make this one optional and leave `endpoints` as required (assuming cue doesn't barf on the change).

Reply:
True, I missed that required field.